.travis.yml

```diff
@@ -5,10 +5,6 @@ dist: jammy
 notifications:
   email: false
 
-env:
-  global:
-    - GO111MODULE=on
-
```

Comment:
Since Go 1.16 the module aware mode is the default behavior so this is not necessary anymore.